frontend/src/pages/home/AlternatingBackgroundColorSectionList.tsx

```diff
@@ -2,45 +2,107 @@ import React from 'react';
 
 import { Grid, useTheme } from '@mui/material';
 
+const SectionGridItem = ({
+  child,
+  color,
+  background,
+  padding,
+}: {
+  child: React.ReactNode;
+  color: string;
+  background: string;
+  padding: string;
+}) => {
+  const theme = useTheme();
+
+  return (
+    <Grid
+      item
+      xs={12}
+      sx={{
+        display: 'flex',
+        justifyContent: 'center',
+        padding: padding,
+        [theme.breakpoints.down('md')]: {
+          p: padding,
+          px: { xs: '16px', md: padding },
+        },
+        color: color,
+        background: background,
+      }}
+    >
+      {child}
+    </Grid>
+  );
+};
+
 const AlternatingBackgroundColorSectionList = ({
   children,
-  secondaryBackground = '#1282B2',
   secondaryColor = 'white',
+  secondaryBackground = '#1282B2',
+  itemPadding = '48px',
+  firstItemPadding = '32px',
+  lastItemIsPrimary,
 }: {
   children: React.ReactNode[];
-  secondaryBackground?: string;
   secondaryColor?: string;
+  secondaryBackground?: string;
+  // Used to wrap every item.
+  itemPadding?: string;
+  // Allow the first item to have a different padding.
+  firstItemPadding?: string;
+  // If true the last item will use the primary color regardless of its
+  // position in the list.
+  lastItemIsPrimary?: boolean;
 }) => {
   const theme = useTheme();
 
+  // children.at(-1) didn't work here...
+  const lastItem = children.slice(children.length - 1);
+
+  const nextColor = (idx: number): string => {
```

Comment:
minor: slightly better names: `colorAtIndex(idx: number)` and `backgroundColorAtIndex(idx: number)`